examples/ros1/src/CMakeLists.txt

```diff
@@ -0,0 +1 @@
+/opt/ros/jade/share/catkin/cmake/toplevel.cmake
```

Comment:
We shouldn't mention this file. First it also does not appear in the folder listing above. Second when building with catkin_make_isolated it is not even there.